<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1778</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>50</y>
     <width>811</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="card_name_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请输入要查询的二手车名称相关信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_name_lineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="Line" name="dead_Line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>800</y>
     <width>1741</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="cards_unit_lineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="cards_unit_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请输入要查询的二手车地区相关信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请输入要查询的二手车价格相关信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_lineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>二手车图片.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="findout_pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>开始查询</string>
   </property>
  </widget>
  <widget class="QTableView" name="card_info_tableView">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>781</width>
     <height>701</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="display_pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>生成可视化图</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_lineEdit_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="price_label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>至</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>万元</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>181</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="goback_pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
